libqtile/layout/base.py

```diff
@@ -320,6 +320,19 @@ def append(self, client):
         """
         self.clients.append(client)
 
+    def standard_add(self, client, position="after_current"):
```

Comment:
Why another method and not merged into `add`?

Reply:
The add method with its numeric argument is used by several layouts.
We want to change the type of this argument for 1 layout (for now).
I think as we are creating something new and different, it is better to separate the old from the new way.
Plus, by adding a new method, it reduces the risk that I insert regressions in the existing method.
TBH, it's also possible that my mind is too set by statically-typed languages.
Let me know if it's not the python way, I'm good to change it.

Reply:
I feel it makes the API more complex (i.e. "why is there two methods for the same thing?") for little gain, when it could be just another argument to the existing method. I'm sure we can make it without regression. :)